packages/ember-template-compiler/lib/plugins/assert-no-each-in.js

```diff
@@ -0,0 +1,41 @@
+import Ember from 'ember-metal/core';
+import { assert } from 'ember-metal/debug';
+import calculateLocationDisplay from 'ember-template-compiler/system/calculate-location-display';
+
+function AssertNoEachIn(options = {}) {
+  this.syntax = null;
+  this.options = options;
+}
+
+AssertNoEachIn.prototype.transform = function AssertNoEachIn_transform(ast) {
+  if (!!Ember.ENV._ENABLE_LEGACY_VIEW_SUPPORT) {
```

Comment:
I just came across https://github.com/emberjs/ember.js/issues/12611. I don't see how I can avoid the `Ember.ENV` right now though